roles/azure_configure_archost/tasks/main.yml

```diff
@@ -0,0 +1,94 @@
+---
+- name: Process tags
+  set_fact:
+    _azure_configure_archost_tags: "{{ _azure_configure_archost_tags | default([]) + [item.key + '=' + item.value] }}"
```

Comment:
I am doing this in my (internal) code for tags:
azure_arc_tags_string: |-
  {%- for k, v in tags_vm.items() %}
  {{ k | quote }}={{ v | replace(",", "\,") | quote }}{% if not loop.last %},{% endif %}
  {%- endfor %}

Reply:
Thanks!